<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppFileSettingsPage</class>
 <widget class="QWidget" name="CppFileSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>503</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>File Naming Conventions</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="headerSuffixLabel">
        <property name="text">
         <string>Header suffix:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="headerSuffixComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sourceSuffixLabel">
        <property name="text">
         <string>Source suffix:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="sourceSuffixComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="lowerCaseFileNamesCheckBox">
        <property name="text">
         <string>Lower case file names</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="licenseTemplateLabel">
        <property name="text">
         <string>License Template:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Utils::PathChooser" name="licenseTemplatePathChooser" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>430</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
